.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Checklist:
+ - Write / update unit tests
+ - Write / update integration (envtest) tests
+ - Ensure there is an issue for e2e tests if needed
+ - Make sure `make test` passes
+ - Test code changes manually
+ - Update readme and documentation
+ - Write PR description that highlights overall changes and add usage examples if applicable
```

Comment:
IMO commit messages provide all the value that PR descriptions do and more (because they're useful for `git log`, git integration in your editor etc., unlike PR descriptions)

The PR description can be a copy of the most relevant commit message or a summary

Reply:
(I'm not saying that the description isn't important, it's very important. I'm saying the description should be in the commit message)

Reply:
This is the thing I wanted to discuss in our retrospective.  I wanted to make sure we agree that we should separate commits where necessary and provide commit message

Reply:
I think it's simple. PR description describes the pull request, not the commit(s). Commit message is where to describe a commit. When there is a single commit inside a PR, the PR description and commit message body are usually same. Both are important.